tests/rules/test_su.py

```diff
@@ -0,0 +1,25 @@
+import pytest
+from thefuck.rules.su import match, get_new_command
+from thefuck.types import Command
+
+
+@pytest.mark.parametrize('output', [
+    'command not found: sudo'])
+def test_match(output):
+    assert match(Command('', output))
+
+
+def test_not_match():
+    assert not match(Command('', ''))
+    assert not match(Command('sudo ls', 'Permission denied'))
+    assert not match(Command('su -c ls', 'Permission denied'))
+    assert not match(Command('ls', 'command not found: ls'))
```

Comment:
`@ pytest.mark.parametrize` is being used in `test_match` for one entry and not in `test_not_match` for four entries. Use it here as well.